docs/topics/analyzer-rbac-known-issue-snippet.adoc

```diff
@@ -0,0 +1,5 @@
+// snippet
+
+.No multi-user access restrictions on resources
+
+There are no multi-user access restrictions on resources. For example, an analyzer task created by a user, can be canceled by any other user. link:https://issues.redhat.com/browse/MTA-4016[(MTA-4016)]
```

Comment:
Shouldn't it be linked to the Jira bug? https://issues.redhat.com/browse/MTA-3819

Reply:
Both snippets LGTM.